packages/@ourworldindata/grapher/src/mapCharts/MapChart.sample.ts

```diff
@@ -25,18 +25,86 @@ export const legacyMapGrapher: GrapherProgrammaticInterface = {
             3512,
             {
                 data: {
-                    years: [2000, 2010, 2010],
-                    entities: [207, 15, 207],
-                    values: [4, 20, 34],
+                    years: [...Array(300).fill(1).map((x,i) => 2000)],
```

Comment:
Changing the sample data here makes some of our tests fail. Can you revert these changes?